AssetDownloader/download.sh

```diff
@@ -1,18 +1,25 @@
 
-DIR_DOWNLODER="$( cd "$(dirname $0)" ; pwd -P )"
+DIR_DOWNLOADER="$( cd "$(dirname $0)" ; pwd -P )"
+cd $DIR_DOWNLOADER
 
 echo " We are currently in the '$DIR_DOWNLOADER' path..."
 
-
-DIR_ROOT="$( cd "$(dirname $0)" ; pwd -P )"
-
+## Back up assers directory
 cd ..
 cd App
 
-rm -R assets
+mv assets assets_bk_`date +%Y_%m_%d_%H%M%S`
 mkdir assets
 
+## Download assets
 cd ..
 cd AssetDownloader
 
 node download.js
+
+## Clean up old assets directories. Leave only the five most recent backups
+cd ..
+while [ $(find App -name assets_bk_* -maxdepth 1 -type d | wc -l) -gt 5 ]
+do
+    find App -name assets_bk_* -maxdepth 1 -type d | sort -n | head -n 1 | xargs rm -rf
+done
```

Comment:
I was a little confused by `cd`ing up and down the file tree. Instead, could we make it so `DIR_DOWNLOADER` is the root?:
```bash
DIR_DOWNLOADER="$( cd "$(dirname $0)" ; pwd -P )"/..
```
and then instead of `cd`ing in and out, we could use it like:
```bash
mv $DIR_DOWNLOADER/App/assets blah blah blah
...
node $DIR_DOWNLOADER/AssetDownloader/download.js
...
find $DIR_DOWNLOADER/App -name assets_bk_* blah blah blah
```

Reply:
Good call. I refactored.